.eslintrc.js

```diff
@@ -9,9 +8,14 @@ module.exports = {
     '@metamask/eslint-config/config/typescript',
   ],
 
+  plugins: [
+    'json',
```

Comment:
This is a bad plugin, but I guess it makes sense to keep using it until we add `prettier` to `@metamask/eslint-config` :disappointed: 
`prettier` completely supplants this. It is faster and better at validating JSON files.

Reply:
Ah, I see. Well, hopefully we should be able to get `prettier` into the base config soon.

In fact, I think `prettier` will replace a number of `@typescript-eslint` formatting rules as well. `@typescript-eslint` has a lot of technical debt and some rules that are straight up broken (e.g. indentation for generics).